### DISCUSSION TITLE
Suppressing Metadata for Viewing

### DISCUSSION THREAD

**Discussion by gsownsby** [photoprism/photoprism#4653] [General]
<img width="319" alt="Screen Shot 2024-12-04 at 4 23 05 PM" src="https://github.com/user-attachments/assets/191fa4af-e4d8-46d3-babb-92f48d64565c">

Hello.  For most of our family's uses of PhotoPrism, showing all of the details of each photo is just a distraction.  How do we turn that off?  Thanks.

**Comment by omerdduran** [comment]
Hi. PhotoPrism does not currently have a built-in setting to completely hide/suppress this metadata from being displayed with photos. If you want, you can use different view modes (such as Mosaic View) that show little metadata

**Reply by gsownsby** [reply]
Thank you for your quick reply.  Perhaps hide/suppress metadata could be considered as a new feature at some point.  Thanks again.

**Reply by omerdduran** [reply]
Hello, I think I found an issue suitable for your request and this issue is currently on the roadmap

- https://github.com/photoprism/photoprism/issues/1164

**Comment by JulianAtkins** [comment]
I'm a first time user of Photoprism. Now that I have it running on my Docker server, I wanted to remove the metadata from underneath the images, but I could not find a way to do this.

Has this feature been implemented? 

Thank you.
